src/zcl/definition/cluster.ts

```diff
@@ -1716,6 +1716,8 @@ const Cluster: {
             acCapacityFormat: {ID: 71, type: DataType.enum8},
             SinopeOccupancy: {ID: 1024, type: DataType.enum8, manufacturerCode: ManufacturerCode.Sinope},
             SinopeBacklight: {ID: 1026, type: DataType.enum8, manufacturerCode: ManufacturerCode.Sinope},
+            StelproSystemMode: {ID: 0x401c, type: DataType.enum8},
```

Comment:
Can we also add `manufacturerCode: ManufacturerCode.Stelpro` here?

Reply:
From what I saw it's not using the manufacturer code there...  but maybe I should check again to be sure.

Reply:
Anyway this mode is not well supported by the converter right now. The thermostat is supporting an _eco_ mode which is not implemented.